doc/man1/dgst.pod

```diff
@@ -8,7 +8,7 @@ dgst, sha, sha1, mdc2, ripemd160, sha224, sha256, sha384, sha512, md4, md5, blak
 
 B<openssl> B<dgst>
 [B<-help>]
-[B<-I<digest>>]
+[B<-[digest]>]
```

Comment:
I always thought that convention is that square brackets denote something you don't have to provide. So with this in mind doesn't suggested modification mean "you have an option to pass dash alone"? Comment applies in multiple cases.